content/actions/reference/environment-variables.md

```diff
@@ -21,15 +21,25 @@ versions:
 To set custom environment variables, you need to specify the variables in the workflow file. You can define environment variables for a step, job, or entire workflow using the [`jobs.<job_id>.steps[*].env`](/github/automating-your-workflow-with-github-actions/workflow-syntax-for-github-actions#jobsjob_idstepsenv), [`jobs.<job_id>.env`](/github/automating-your-workflow-with-github-actions/workflow-syntax-for-github-actions#jobsjob_idenv), and [`env`](/github/automating-your-workflow-with-github-actions/workflow-syntax-for-github-actions#env) keywords. For more information, see "[Workflow syntax for {% data variables.product.prodname_dotcom %}](/articles/workflow-syntax-for-github-actions/#jobsjob_idstepsenv)."
 
 ```yaml
-steps:
-  - name: Hello world
-    run: echo Hello world $FIRST_NAME $middle_name $Last_Name!
+jobs:
+  weekday_job:
+    runs-on: ubuntu-latest
     env:
-      FIRST_NAME: Mona
-      middle_name: The
-      Last_Name: Octocat
+      DAY_OF_WEEK: Mon
+    steps:
+      - name: "Hello world when it's Monday"
+        if: env.DAY_OF_WEEK == 'Mon'
+        run: echo "Hello world $FIRST_NAME $middle_name $Last_Name, sounds like you have a case of the Mondays!"
```

Comment:
I'd suggest replacing this colloquialism, to help with translation/accessibility:
```suggestion
        run: echo "Hello world $FIRST_NAME $middle_name $Last_Name, today is Monday!"
```

Reply:
For posterity; @martin389 and I had a robust discussion about the _Office Space_ reference in this example. But, eventually we concluded that especially for people are reading this example not in English (or if English isn’t their first language), this might be confusing if they have never seen the movie. 🙁

The docs must come first, even if this was an awesome reference. 😢